custom-connectors/PUGGamifiedEngagement/apiDefinition.swagger.json

```diff
@@ -0,0 +1,681 @@
+                    }
+                  },
+                  "description": "player"
+                }
+              }
+            }
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        }
+      }
+    },
+    "/api/players/{player_id}/currency_accounts": {
+      "get": {
+        "summary": "Get player accounts",
+        "description": "Get player Accounts",
+        "operationId": "Get_Player_Accounts",
+        "parameters": [
+          {
+            "name": "player_id",
+            "in": "path",
+            "required": true,
+            "type": "integer"
+          },
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": " application/vnd.sno-ge.iapi+json",
+            "description": "Content-Type",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "Accept",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": " application/vnd.sno-ge.iapi+json",
+            "description": "Accept",
+            "x-ms-visibility": "internal"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        }
+      }
+    },
+    "/api/players/{player_id}/currency_accounts/{account_type}": {
+      "get": {
+        "summary": "Get account balance",
+        "description": "Get account balance",
+        "operationId": "Get_Account_Balance",
+        "parameters": [
+          {
+            "name": "player_id",
+            "in": "path",
+            "required": true,
+            "type": "integer"
+          },
+          {
+            "name": "account_type",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": " application/vnd.sno-ge.iapi+json",
+            "description": "Content-Type",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "Accept",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": " application/vnd.sno-ge.iapi+json",
+            "description": "Accept",
+            "x-ms-visibility": "internal"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        }
+      }
+    },
+    "/api/players/{player_id}/currency_accounts/{account_type}/add_currency/{amount}": {
+      "post": {
+        "summary": "Add Points",
+        "description": "Add Points",
```

Comment:
@smart-jp Description values are pretty vague, Can you please provide some information of the operation.